controller/api/views.py

```diff
@@ -17,6 +19,19 @@
 from api import authentication, models, permissions, serializers, viewsets
 
 
+class HealthCheckView(View):
+    """Simple health check view to determine if the server
+       is responding to HTTP requests.
+    """
+
+    def get(self, request):
+        return HttpResponse("OK")
+    head = get
+
+# So urlconf doesn't have to import this view
+health_check = HealthCheckView.as_view()
```

Comment:
I think we should still get urlconf to import this view, so we have it all in one place rather than multiple places.